app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
     <string name="title_remember">Remember</string>
     <string name="title_export">Export</string>
     <string name="title_import">Import</string>
-    <string name="title_view_only_mode">View-only Mode</string>
-    <string name="title_hide_remote_cursor">Hide Remote Cursor</string>
-    <string name="title_key_compat_mode">Keyboard Compatibility Mode</string>
-    <string name="title_use_ssh_tunnel">Use SSH Tunnel</string>
-    <string name="title_unknown_ssh_host">Unknown SSH Host</string>
-    <string name="title_ssh_host_key_changed">Host Key changed!</string>
+    <string name="title_view_only_mode">View-only</string>
+    <string name="title_hide_remote_cursor">Hide remote cursor</string>
+    <string name="title_key_compat_mode">Keyboard compatibility mode</string>
+    <string name="title_use_ssh_tunnel">SSH tunnel</string>
+    <string name="title_unknown_ssh_host">Unknown SSH host</string>
+    <string name="title_ssh_host_key_changed">The host key changed!</string>
     <string name="title_continue">Continue</string>
-    <string name="title_delete_servers_before_import">Delete current Servers before Import</string>
+    <string name="title_delete_servers_before_import">Remove all servers before importing</string>
```

Comment:
Should we change `all` to `previous`/`old`, something that denotes currently saved servers?